src/components/Loader/index.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Loader } from './index';
+import { shallow } from 'enzyme';
+
+describe('Loader', () => {
+  let wrapper;
+
+  it('should match snapshot', () => {
+    wrapper = shallow (
+      <Loader />
+    )
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
Sweet, tested and everything! Also I didn't mean to click start a review....sometimes I swear I don't know how to GitHub...

Reply:
Haha, no big deal! Thanks for clarifying what happened there!